tests/braille-specs/de-g0-detailed-specs.yaml

```diff
@@ -211,18 +211,18 @@ tests:
 
 # Capitals
   - ["AUA", "⠘⠁⠥⠁"]
-  - ["ČSSR", "⠘⠈⠉⠎⠎⠗", {xfail: "Backtranslates as cents"}]
+  - ["ČSSR", "⠘⠈⠉⠎⠎⠗", xfail: {backward: "Backtranslates as cents"}]
   - ["U-Bahn", "⠘⠥⠤⠨⠃⠁⠓⠝"]
   - ["kW kVA hPa", "⠅⠘⠺ ⠅⠘⠧⠁ ⠓⠨⠏⠁"]
-  - ["GmbH ICs KPdSU", "⠨⠛⠍⠃⠘⠓ ⠘⠊⠉⠠⠎ ⠘⠅⠏⠠⠙⠘⠎⠥", {xfail: "BT as '"}]
+  - ["GmbH ICs KPdSU", "⠨⠛⠍⠃⠘⠓ ⠘⠊⠉⠠⠎ ⠘⠅⠏⠠⠙⠘⠎⠥", xfail: {backward: "BT as '"}]
 
 
 # "correct" rules
-  - [" -1", " ⠈⠤⠼⠁", {xfail: "FT adds space"}]
+  - [" -1", " ⠈⠤⠼⠁", xfail: {backward: true}]
   - ["/-4abc- ", "⠐⠂⠤⠼⠙⠠⠁⠃⠉⠤ "]
```

Comment:
The old comment says forward and mentions a reason "adds space"

Reply:
Yes but forward passes and backward fails, and not because of spaces. The comment was wrong.

Reply:
OK